docs/release_notes/v0.9.0.md

```diff
@@ -0,0 +1,331 @@
+  
+# Dapr 0.9.0
+
+We're happy to announce the release of Dapr 0.9.0!
+
+**Highlights**
+
+
+* Added new components: [Scheduler binding](https://github.com/dapr/docs/blob/master/reference/specs/bindings/cron.md), [Postgresql state store](https://github.com/dapr/docs/tree/master/howto/setup-state-store/setup-postgresql.md)
+* Dapr dashboard Helm integration
+* Command Completion feature in Dapr CLI
```

Comment:
Consistent language, we either use verbs `Added` or just list of the areas that were impacted `Command completion in CLI`

Reply:
^ yes, I realize how ironic it is that I for all people comment about gramma ;)

Reply:
:) Fixed